openedx/features/content_tagging/rules.py

```diff
@@ -11,14 +11,16 @@
 User = get_user_model()
 
 
-def is_taxonomy_admin(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
+def is_taxonomy_user(user: User, taxonomy: oel_tagging.Taxonomy = None) -> bool:
     """
-    Returns True if the given user is a Taxonomy Admin for the given content taxonomy.
+    Returns True if the given user is a Taxonomy User for the given content taxonomy.
 
-    Global Taxonomy Admins include global staff and superusers, plus course creators who can create courses for any org.
-    Otherwise, a taxonomy must be provided to determine if the user is a org-level course creator for one of the
-    taxonomy's org owners.
+    Taxonomy users include global staff and superusers, plus course creators who can create courses for any org.
+    Otherwise, we need a taxonomy provided to determine if the user is an org-level course creator for one of the
+    orgs allowed to use this taxonomy.
     """
+    # ToDo: This is a temporary fix to not break can_change_taxonomy_tag and can_change_object_tag
+    # Should be revised when the API that uses these roles is implemented
```

Comment:
Could you remove these lines? We don't need to excuse this method anymore; it's used by `can_change_object_tag`, and that's fine.
```suggestion
```